packages/sls-env/src/index.ts

```diff
@@ -34,7 +34,7 @@ import type {
 type SlsEnvironment<H extends Handler<any, any, any>, C, D, P = HandlerPayload<H>, R = ReturnType<H>> = {
   errorHandler: (handler: ErrorHandler<ReturnType<H>>) => SlsEnvironment<H, C, D, P, R>
   successHandler: (handler: SuccessHandler<R, ReturnType<H>>) => SlsEnvironment<H, C, D, P, R>
-  global: (dependencies: DependenciesConstructor<C, D>) => SlsEnvironment<H, C, D, P, R>
+  global: (dependencies: DependenciesConstructor<C, P, D>, config?: DependenciesConfig) => SlsEnvironment<H, C, D, P, R>
```

Comment:
why do we need a payload in dependencies construct?

`dependencies` construct should not depend on payload as they are `global` and cached through lifecycle of `lambda`?